ACPs/23-p-chain-native-transfers.md

```diff
@@ -0,0 +1,58 @@
+```text
+ACP: <PR Number>
+Title: P-chain Native Transfers
+Author(s): Dhruba Basu <https://github.com/dhrubabasu>
+Discussions-To: <GitHub Discussion URL>
+Status: Proposed
+Track: Standards
+```
+
+## Abstract
+
+Support native transfers on P-chain. This enables users to transfer P-chain assets without leaving the P-chain or using a transaction type that's not meant for native transfers.
+
+## Motivation
```

Comment:
You should add 1 extra paragraph here that highlights how the growth of elastic subnets on the P-Chain could lead to much wider usage of transfers and that it would be a good idea to get ahead of it.